src/metabase/search/config.clj

```diff
@@ -22,7 +22,7 @@
 
 (def ^:const max-filtered-results
   "Number of results to return in an API response"
-  50)
+  125)
```

Comment:
@kdoh @howonlee it looks like due to performance reasons we will have 2.5 pages at a maximum which does not bring much value. Can we go with a higher number here or maybe we should just render all 125 items without pagination?

Reply:
I think the pagination was all for UX reasons not perf

I tried 1000 with 1000 search reusults and that was annoyingly slow, so I just pulled 125 out of my butt, it's not like a principled rigorous profile or anything

Reply:
Sorry I may be a bit confused, what is this number doing here? And just for proper context, our primary motivation for adding pagination around the app is definitely for performance reasons.

Reply:
@kdoh even with pagination, we have to limit the total number of matched items due to performance reasons. No matter if we request only the first page with 50 of 10000 items it is still slow on the back end. Is that correct, @howonlee?

Reply:
Got it. I'd see if we could up this limit a bit then.